client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -147,6 +148,19 @@ export default React.createClass( {
 		return {};
 	},
 
+	getDefaultActionLinks( plugin ) {
+		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
```

Comment:
Uh, nor does it seem to get a `sites` prop, see https://github.com/Automattic/wp-calypso/blob/9fa4f44304e7c5d04edbd4a73b46dda6b9a1a3dd/client/my-sites/plugins/plugin-meta/index.jsx#L473-L480